projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -847,20 +853,21 @@ private function get_integration( $slug ) {
 
 		// Base shape for all integrations.
 		$base = array(
-			'id'              => $slug,
-			'slug'            => $slug,
-			'type'            => $type,
-			'title'           => isset( $config['title'] ) ? sanitize_text_field( $config['title'] ) : '',
-			'subtitle'        => isset( $config['subtitle'] ) ? sanitize_text_field( $config['subtitle'] ) : '',
-			'marketingUrl'    => $marketing_redirect_slug ? Redirect::get_url( $marketing_redirect_slug ) : null,
-			'pluginFile'      => ( $type === 'plugin' && ! empty( $config['file'] ) ) ? str_replace( '.php', '', $config['file'] ) : null,
-			'isInstalled'     => false,
-			'isActive'        => false,
-			'needsConnection' => ( $type === 'service' ),
-			'isConnected'     => false,
-			'version'         => null,
-			'settingsUrl'     => null,
-			'details'         => array(),
+			'id'               => $slug,
+			'slug'             => $slug,
+			'type'             => $type,
+			'title'            => isset( $config['title'] ) ? sanitize_text_field( $config['title'] ) : '',
+			'subtitle'         => isset( $config['subtitle'] ) ? sanitize_text_field( $config['subtitle'] ) : '',
+			'marketingUrl'     => $marketing_redirect_slug ? Redirect::get_url( $marketing_redirect_slug ) : null,
+			'pluginFile'       => ( $type === 'plugin' && ! empty( $config['file'] ) ) ? str_replace( '.php', '', $config['file'] ) : null,
+			'isInstalled'      => false,
+			'isActive'         => false,
+			'needsConnection'  => ( $type === 'service' ),
+			'isConnected'      => false,
+			'version'          => null,
+			'settingsUrl'      => null,
+			'details'          => array(),
+			'enabledByDefault' => isset( $config['enabled_by_default'] ) ? (bool) $config['enabled_by_default'] : false,
```

Comment:
Only change here is adding the enabledByDefault item at the bottom. Rest is spacing.